Added all tests for Radixsort (incl edge cases)

babyygemperor opened the pull request:
Except for the final sort

**Shortly describe your proposed changes:**
 * None


**Link any issues if they have led to changes in the request:**
 * None


**Note the (sub-)exercises the tests are meant for:**

* GAD 07 - RadixSort

babyygemperor commented:
Alright, all tests are here, lemme know when something's off

N0W0RK reviewed (changes requested):
Could you create two different PRs for Exercise 6 and 7. I could go ahead and check the changes for 6 but I havent yet had the tim to look at exercise 7

babyygemperor commented:
Oh shit I didn't notice that my issue fix commit got committed to this branch, lemme try to fix that

babyygemperor commented:
I'm still waiting for someone to approve the changes to this pull request. It's been a week now

CrsiX commented:
You didn't eliminate the changes introduced for exercise 6 yet, as outlined by @N0W0RK. After reverting your changes on those files, I guess he will also agree on this PR (and the style checker wouldn't complain about it anymore) :)

I did not add a review on purpose, because I can't guarantee that those tests are correct since I haven't started the exercise yet. The only thing I could say would be "yes, it compiles". That's not meaningful enough to use in a code review :)

N0W0RK commented on GAD 7 - Radixsort/tests/radix/FastResult.java:
```diff
@@ -0,0 +1,25 @@
+package tests.radix;
+
+import java.util.ArrayList;
```
As far as I can tell there should be an import for gad.radix.Result here

babyygemperor replied:
Fixed

N0W0RK commented on GAD 7 - Radixsort/tests/radix/RadixsortTester.java:
```diff
@@ -0,0 +1,821 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class RadixsortTester {
```
The class should be named RadixSortTester

babyygemperor replied:
Renamed

N0W0RK reviewed (changes requested):
When you fix the two changes I added and remove the GAD 6 files from this PR I will approve this.

babyygemperor commented:
I've fixed all the issues and addressed all the code style errors which I could.
I've left the expected results in a bracket, so that it's easier to just collapse the entire hard coded result and easily be able to navigate through the tests. The line length limitation is albeit important, but I don't know how I should address that issue
